### ISSUE TITLE
EADDRINUSE

### ISSUE THREAD

**Issue by arturi** [evbogue/minbay#1]
Hi! Thanks for writing minbay, excited to try it out. Still getting this:

```
artur at AMBP in ~/Desktop/minbay on master [!]
$ npm start

> minbay@1.3.0 start /Users/artur/Desktop/minbay
> node sbot server --allowPrivate

Log level: notice
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE :::3000
    at Object.exports._errnoException (util.js:1022:11)
    at exports._exceptionWithHostPort (util.js:1045:20)
    at Server._listen2 (net.js:1259:14)
    at listen (net.js:1295:10)
    at Server.listen (net.js:1391:5)
    at Object.exports.serve (/Users/artur/Desktop/minbay/serve.js:8:5)
    at Object.<anonymous> (/Users/artur/Desktop/minbay/sbot.js:63:8)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
```

even after trying everything I read on scuttlebut, commenting out line 52 and the like. Help appreciated :) Thanks!

**Comment by evbogue** [comment]
@arturi Thanks for trying Minbay! 

This error is because you have two copies of `ssb-ws` running, as one is bundled with minbay. The other `ssb-ws` you probably installed in `.ssb/plugins`. A few people have reported this, so I think I'll go ahead and push a temporary fix of the problem. 

The reason `ssb-ws` is bundled is so you don't have to install a bunch of ssb plugins to get started with minbay.

I disabled minbay's loading of the `.ssb/plugins` folder. Do a pull, and let me know if this error goes away!

**Comment by evbogue** [comment]
oh wait, this is port 3000, not port 8989. Are you running a server at port 3000 somewhere on your computer?

**Comment by evbogue** [comment]
@arturi Try restarting your computer or running `sudo killall node` in your terminal. You might have another copy of minbay still running on your system that didn't shut down properly?

**Comment by arturi** [comment]
Yes, I was, as it turned out, thanks :) Maybe better to go with something like 6734 here? Or 3279 :)

**Comment by arturi** [comment]
Works now! Thank you!

**Comment by evbogue** [comment]
@arturi How about 3013? I'll change and push.

**Comment by arturi** [comment]
Great, thanks!

**Comment by evbogue** [comment]
Fixed in 653b63fa81d74167101576295e54299f0b9b9cad
